source/module_elecstate/elecstate_getters.cpp

```diff
@@ -9,4 +12,39 @@ double get_ucell_omega()
     return GlobalC::ucell.omega;
 }
 
+std::string get_input_vdw_method()
```

Comment:
Is it better to define it as an inline function?

Reply:
> Is it better to define it as an inline function?

Seems not, because one wouldl not be able to break dependencies on the INPUT object